src/benchmark_environments/testing/envs.py

```diff
@@ -184,3 +184,40 @@ def test_premature_step(env: gym.Env, skip_fn, raises_fn) -> None:
     act = env.action_space.sample()
     with raises_fn(Exception):  # need to call env.reset() first
         env.step(act)
+
+
+class CountingEnv(gym.Env):  # pragma: no cover
+    """At timestep `t` of each episode, has `t == obs == reward / 10`.
+
+    Episodes finish after `episode_length` calls to `step()`, or equivalently
+    `episode_length` actions. For example, if we have `episode_length=5`,
+    then an episode has the following observations and rewards:
+
+    ```
+    obs = [0, 1, 2, 3, 4, 5]
+    rews = [10, 20, 30, 40, 50]
+    ```
+    """
+
+    def __init__(self, episode_length=5):
```

Comment:
maybe type annotations? (I'd probably have written it with them, I'm not sure if it's worth changing now though, it's only code used in tests.)